src/store/actions/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export */
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import * as types from '../types';
+
+const URL = 'https://api.rawg.io/api';
+
+export const getGames = createAsyncThunk(types.GET_GAMES,
+  async () => {
+    const { data } = await axios.get(`${URL}/games?key=b83701e00bdf4e8fa7e7e7c8f6c1d9cb`);
```

Comment:
- [Optional] It's very important to secure your API key. Some solutions provided [here](https://stackoverflow.com/questions/48699820/how-do-i-hide-api-key-in-create-react-app) would help you to secure your API key.